packages/web-sdk/src/core/sync.ts

```diff
@@ -322,14 +326,18 @@ export class SyncManager {
             if (request.payloadId) {
                 await (this.context.client as any)[request.action](
                     request.payloadId,
-                    { input: request.payload }
```

Comment:
@AfeezGL Nice catch! I was testing the retry queue but forgot to revert this change. Thank you